docs/customer.rb

```diff
@@ -0,0 +1,18 @@
+class Customer
+  attr_reader :customer_order_array
+
+  def initialize
+    @customer_order_array = []
+  end
+
+  def customer_order
+    p "What meal would you like: "
+    user_meal_input = gets.chomp
+    p '----------------------'
+    p "How many of those would you like?: "
+    user_amount_input = gets.chomp
+    p '-----------------------'
+    @customer_order_array << { :customer_meal => user_meal_input,
+      :amount_ordered => user_amount_input }
+  end
```

Comment:
This could be refactored to take in two arguments into the customer order method so that user_meal_input is set to one of the arguments and user_amount_input could be set to the other one so that means when you call the method you can declare what you want to order and how much u want to order in one go